.github/workflows/publish-workspace-helm-chart.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish workspace helm chart
+
+on:
+  repository_dispatch:
+    types: [ publish-workspace-helm-chart ]
+
+permissions:
+  id-token: write # This is required for requesting the JWT
+  packages: write
+  contents: write
+  actions: read
+  deployments: read
+  pull-requests: read
+
+jobs:
+  publish-workspace-helm:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@0634a2670c59f64b4a01f0f96f84700a4088b9f0 # v2.12.0
+        with:
+          egress-policy: audit
+          disable-sudo: true
+          disable-telemetry: true
+
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          submodules: true
+          fetch-depth: 0
+          ref: ${{ github.event.client_payload.tag }}
+
+      - name: Publish workspace helm chart
+        uses: stefanprodan/helm-gh-pages@0ad2bb377311d61ac04ad9eb6f252fb68e207260 # v1.7.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          charts_dir: charts/kaito/workspace
+          target_dir: charts/kaito
+          linting: off
```

Comment:
It seems that `create release` is not triggered after helm chart published

Reply:
added some handling to overwrite the values.yaml with the rag service image info